graph.js

```diff
@@ -133,17 +152,39 @@ app
     }
   })
 
+/**
+ * graph auth */
+app
+  .command('auth [NODE] [AUTH_TOKEN]')
+  .description("Stores auth token on the system's keychain for use by deploy")
+  .action((node, authToken) => {
+    if (
+      authToken === undefined ||
+      node === undefined ||
+      authToken > 200
+    ) {
+      outputAuthConfig(node, authToken)
+      console.error('--')
+      console.error('For more information run this command with --help')
+      process.exitCode = 1
+      return
+    }
+    keytar.setPassword('subgraph-hosting', node, authToken).then(() => {
+      console.log("Auth token saved on keychain for Graph node:", node)
+    }).catch((err) => {
+      console.error('Failed to save auth token to the keychain: ', err)
+    })
+  })
+
+
 /**
  * graph deploy
  */
 app
   .command('deploy [SUBGRAPH_MANIFEST]')
   .description('Deploys the subgraph to a graph node')
-  .option(
-    '-k, --api-key <KEY>',
-    'Graph API key authorized to deploy to the subgraph name'
-  )
   .option('-g, --node <URL>[:PORT]', 'Graph node to deploy to')
+  .option('-a, --auth', 'Use auth token for deployment')
```

Comment:
That's not how it's supposed to work. First of all, let's not introduce a short `-a` flag for now. The long flag should be `--access-token <ACCESS_TOKEN>`, as a way to override the (potentially) saved access token for a node.